synfig-studio/images/CMakeLists.txt

```diff
@@ -230,8 +230,13 @@ set(IMAGES
     ${ICONS}
 )
 
+set(PIXMAPS_DIR ${SYNFIG_BUILD_ROOT}/share/pixmaps/synfigstudio)
+
 ## TODO: don't run this if files are up to date
-add_custom_target(build_images)
+add_custom_target(build_images
+    COMMAND ${CMAKE_COMMAND} -E make_directory ${PIXMAPS_DIR}
```

Comment:
Different CMakeLists.txt files have different minimum version. 
```
./ETL/CMakeLists.txt:cmake_minimum_required(VERSION 2.8)
./CMakeLists.txt:cmake_minimum_required(VERSION 2.8)
./synfig-studio/src/CMakeLists.txt:cmake_minimum_required(VERSION 3.1)
./synfig-studio/CMakeLists.txt:cmake_minimum_required(VERSION 3.1)
./synfig-core/src/CMakeLists.txt:cmake_minimum_required(VERSION 3.1)
./synfig-core/CMakeLists.txt:cmake_minimum_required(VERSION 3.1)
```
We should update the version for those 2 above and probably should also make it consistent with other, either by being on `3.1` or `3.0`. It is kinda off topic-y but still.

 And I will update it so that it uses `file(MAKE_DIRECTORY)` instead of `make_directory`.

Reply:
Didn't make any changes on minimum version yet, but fixed the other part.